Dockerfile

```diff
@@ -14,5 +14,9 @@ ENV BIGCHAINDB_CONFIG_PATH /data/.bigchaindb
 ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 ENV BIGCHAINDB_WSSERVER_HOST 0.0.0.0
 ENV BIGCHAINDB_WSSERVER_SCHEME ws
+
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME ws
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_HOST 0.0.0.0
```

Comment:
I don't understand why all these BIGCHAINDB_WSSERVER_* environment variables are being set in the Dockerfile. If the environment variable is set, it overrides the default value, and any value in a local config file.

Why isn't BIGCHAINDB_WSSERVER_PORT set?